README.md

```diff
@@ -25,179 +25,79 @@ This MCP server provides access to Trading Simulator operations through structur
-
-Example:
-```
-Authorization: Bearer your_api_key_here
-```
-
 ## Setup
 
 1. Clone the repository
-   ```bash
-   git clone https://github.com/yourusername/trading-simulator-mcp.git
-   cd trading-simulator-mcp
-   ```
-
-2. Install dependencies
+2. Install dependencies:
    ```bash
    npm install
    ```
-
-3. Configure your API credentials (see Configuration section below)
-
-4. Build the project
+3. Build the project:
    ```bash
    npm run build
    ```
 
-5. Start the server
-   ```bash
-   npm run start
-   ```
-
-## Configuration
-
-You have two options for configuring the Trading Simulator MCP server:
-
-### Method 1: Direct Configuration in Cursor/Claude (Recommended)
-
-The recommended approach is to provide environment variables directly in your Cursor or Claude Desktop configuration. This is more secure and eliminates the need for a .env file.
-
-- The server will automatically use these environment variables when provided through the configuration.
-- See the "Adding to Cursor" and "Adding to Claude Desktop" sections below for specific setup instructions.
+## Required Environment Variables
 
-### Method 2: Using a .env File (Fallback)
+The The Trading Simulator MCP requires the following environment variables:
 
-If you prefer to use a .env file, or are running the server directly from the command line:
-
-1. Create a `.env` file with your API credentials
-   ```bash
-   cp .env.example .env
-   ```
-
-2. Edit the `.env` file with your API key
-   ```
-   TRADING_SIM_API_KEY=your_api_key_here
-   TRADING_SIM_API_URL=http://localhost:3000
-   DEBUG=false
-   ```
-
-3. Secure your .env file with restricted permissions
-   ```bash
-   chmod 600 .env
-   ```
+```
+TRADING_SIM_API_KEY=your-api-key
+```
 
-## Environment Variable Precedence
+## Usage
 
-The Trading Simulator MCP server uses the following order of precedence for environment variables:
+### Running Locally
 
-1. Environment variables provided directly from JSON configuration
-2. Environment variables from a .env file (if present and #1 is not available)
-3. Default values for optional variables (e.g., API_URL defaults to "http://localhost:3000")
+```bash
+# Start the server with environment variables
+TRADING_SIM_API_KEY=your-api-key npm start
+```
 
-## Adding to Cursor
+### Adding to Cursor, Claude, etc.
 
 To add this MCP server to Cursor:
 
-1. Build the project first with `npm run build`
-2. In Cursor, go to Settings > MCP Servers
-3. Click "Add Server"
-4. Configure the server with the following settings:
+1. Configure the server with the following settings:
    - **Name**: `Trading Simulator MCP` (or any name you prefer)
    - **Type**: `command`
    - **Command**: `node`
-   - **Arguments**: `/path/to/trading-sim-mcp/dist/index.js` (use the full path)
+   - **Arguments**: `/path/to/trading-simulator-mcp/dist/index.js` (replace with your actual path)
```

Comment:
If the whole npx flow is indeed working, then let's just document that instead.

Reply:
sounds good - left only npx instructions: [eb2bf5c](https://github.com/recallnet/trading-simulator-mcp/pull/3/commits/eb2bf5c033c81ecaedfbfde31ef15d16727bcb2b)